src/main/java/mate/academy/Main.java

```diff
@@ -1,7 +1,18 @@
 package mate.academy;
 
+import java.math.BigDecimal;
+import mate.academy.dao.BookDao;
+import mate.academy.lib.Injector;
+import mate.academy.model.Book;
+
 public class Main {
-    public static void main(String[] args) {
+    private static final Injector injector = Injector.getInstance("YOUR_PACKAGE");
```

Comment:
`YOUR_PACKAGE` should be changed to YOUR package. 
e.g. mate.academy

P.S. It is always the same for the same project. We wont change the package name or structure.

```suggestion
    private static final Injector injector = Injector.getInstance("YOUR_PACKAGE");
```